src/main/java/org/bytedeco/javacpp/Pointer.java

```diff
@@ -431,6 +431,7 @@ public static long parseBytes(String string) throws NumberFormatException {
         }
         long size = Long.parseLong(string.substring(0, i));
         switch (string.substring(i).trim().toLowerCase()) {
+            case "%": size = size * relativeUnit * 1024L / 100; break;
```

Comment:
We're actually providing the multiple here, so no need to multiply it with anything else. Maybe that would be a better name for the variable, `relativeMutliple`? What about adding a unit test for that?

Reply:
I just added a test and that helped clear up the algorithm error. Let me know what you think of it.